database/scripts/test_regression_history.sql

```diff
@@ -0,0 +1,13 @@
+SELECT test_failures.error_name,
+    test_failures.job_name,
```

Comment:
The sql is alright, but I think the name of the should reflect your usage of the param1 and param2, with "test_regression_history" it's not straightforward to me what this sql should do.

Reply:
WDYT about: `error_appearances_in_job`?

Reply:
SGTM!